lib/ibm/cloud/sdk_http/base_collection.rb

```diff
@@ -83,17 +84,33 @@ def create(payload, payload_type = 'json')
 
         private
 
+        # Return a wrapped instnace if set.
+        # @param value [Hash] The hash returned from server.
+        def hash_instance(value)
+          return @instance.new(self, data: value, id_key: @instance_id) if @instance
+
+          value
+        end
+
+        # Get without any additional logic.
+        # @param url [String] Full URL to send to server.
+        def adhoc_get(url)
+          @connection.request('get', url, metadata(@params)).raise_for_status?.json
```

Comment:
Just noticed the `raise_for_status?`...not sure that was there previously or not, but it's unexpected (and not idiomatic Ruby) to have a predicate method (i.e ending with a `?`) return anything other than a boolean, and generally you don't chain off of a predicate method.  If the method actually raises, you may want to use a `!` at the end of the method instead.

Reply:
Oh okay. That can be changed.

Reply:
Changed